README.md

```diff
@@ -1,153 +1,108 @@
 [![License](https://img.shields.io/badge/license-Apache--2.0-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0)
 [![Join Slack](https://img.shields.io/badge/join-slack-9B69A0.svg)](https://nimbella-community.slack.com/)
 [![Twitter](https://img.shields.io/twitter/follow/nimbella.svg?style=social&logo=twitter)](https://twitter.com/intent/follow?screen_name=nimbella)
 
-A bunch of command-sets that can be installed using [Nimbella Commander](https://nimbella.com/product/commander) to run on messaging platforms like Slack (<img alt="slack"  width="13" height="13" src="https://unpkg.com/simple-icons@latest/icons/slack.svg" />) and Mattermost (<img alt="mattermost"  width="13" height="13" src="https://unpkg.com/simple-icons@latest/icons/mattermost.svg" />).
+[Nimbella Commander](https://nimbella.com/product/commander) is the best, fastest and easiest way to build bots and commands for messaging platforms such as [Slack](https://slack.com/apps/AS833QXL0-nimbella-commander), Microsoft Teams, and Mattermost. Commander requires no new accounts to setup, no command line tools to start, and you will create your first command in seconds. The development experience can extend from your browser, to your terminal, so that you can develop, test, run and integrate bots and commands where it is most convenient for your workflow.
+
+Unique features of Nimbella Commander:
+
+1. **Command Sets:** a convenient way to bundle, share, and install commands into your messaging platform.
+2. **Access Control:** control which users can run, edit, or administer commands
+3. **Secrets Management:** securly bind API keys to commands
+4. **Audit Logs and Operator Dashboard:** monitor and administer usage.
+
+Learn more on how Commander can help you by clicking [here](https://nimbella.com/resources-commander/overview#what-is-commander).
+
+- [Installation instructions](#Installation)
+  - [Slack](#Slack) (<img alt="slack"  width="13" height="13" src="https://unpkg.com/simple-icons@latest/icons/slack.svg" />)
+  - [Teams](#Teams) (<img alt="mattermost"  width="13" height="13" src="https://unpkg.com/simple-icons@latest/icons/microsoftteams.svg" />) (coming soon)
+  - [Mattermost](#Mattermost) (<img alt="mattermost"  width="13" height="13" src="https://unpkg.com/simple-icons@latest/icons/mattermost.svg" />) (coming soon)
+- [Command Sets](#Command-Sets)
+  - [Installing a command set](#Installing-a-Command-Set)
+  - [Catalog of command sets](#Catalog-of-Command-Sets)
+- [Support](#support)
+- [License](#license)
+- Additional resources
+  - [Commander reference manual](https://nimbella.com/resources-commander/reference)
 
 ## Installation
 
-Make sure you've Nimbella Commander installed. If not, click [here](https://slack.com/oauth/authorize?client_id=876870346995.892105847680&scope=commands) to install it on your slack workspace.
```

Comment:
Add short intro Para here as I. The installation instructions for commander depend on the platform used  select the one that works for you.